server/src/schema.js

```diff
@@ -32,36 +37,40 @@ const schema = new graphql.GraphQLSchema({
       measurement: {
         type: types.measurementType,
         args: _.pick(attributeFields(models.Measurement), ['id']),
-        resolve: resolver(models.Measurement),
+        resolve: resolvers.measurements.one,
       },
+
       measurements: {
         type: new graphql.GraphQLNonNull(graphql.GraphQLList(types.measurementType)),
-        resolve: resolver(models.Measurement),
+        args: {
+          measurableId: { type: graphql.GraphQLNonNull(graphql.GraphQLString) },
+        },
+        resolve: resolvers.measurements.all,
       },
 
       measurable: {
         type: types.measurableType,
         args: _.pick(attributeFields(models.Measurable), ['id']),
-        resolve: resolver(models.Measurable),
+        resolve: resolvers.measurables.one,
       },
+
       measurables: {
         type: new graphql.GraphQLNonNull(new graphql.GraphQLList(types.measurableType)),
         args: {
           offset: { type: graphql.GraphQLInt },
           limit: { type: graphql.GraphQLInt },
           creatorId: { type: graphql.GraphQLString },
-          channelId: { type: graphql.GraphQLString },
           seriesId: { type: graphql.GraphQLString },
+          channelId: { type: graphql.GraphQLNonNull(graphql.GraphQLString) },
```

Comment:
Can this stay nullable? It would be nice if a user could call it without channelId, and then it figures out which channels that user has access to, then gathers all the measurables.

Reply:
Ok, will fix.

Reply:
Fixed.